packages/angular/src/generators/change-storybook-targets/change-storybook-targets.ts

```diff
@@ -8,13 +12,20 @@ export async function angularChangeStorybookTargestGenerator(
   tree: Tree,
   schema: Schema
 ) {
-  ensurePackage(tree, '@nrwl/storybook', nxVersion);
-  const { changeStorybookTargetsGenerator } = await import('@nrwl/storybook');
+  const { changeStorybookTargetsGenerator } = ensurePackage(
+    '@nrwl/storybook',
+    nxVersion
+  );
   await changeStorybookTargetsGenerator(tree);
 
   if (!schema.skipFormat) {
     await formatFiles(tree);
   }
+  return addDependenciesToPackageJson(
+    tree,
+    {},
+    { '@nrwl/storybook': nxVersion }
+  );
```

Comment:
I'm unsure whether it is an issue, but better to leave formatting as the last thing to do. We don't have to double-guess or rely on whether the `addDependenciesToPackageJson` make changes with the right formatting:
```suggestion
  const installTask = addDependenciesToPackageJson(
    tree,
    {},
    { '@nrwl/storybook': nxVersion }
  );

  if (!schema.skipFormat) {
    await formatFiles(tree);
  }

  return installTask;
```